static/search.js

```diff
@@ -674,13 +674,45 @@ var Search = Control.extend({
 	// function activateResult
 	// sets property and adds class to active result
 	activateResult: function($result){
+		// Get position top of last active element
+		var lastResultPosTop = parseInt(this.$activeResult && this.$activeResult.position().top || 0);
```

Comment:
Make sure to always pass `10` as the second argument to `parseInt` (you can search for reasons why).